pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -651,6 +649,17 @@ func (h *Helper) ServerVersion() (semver.Version, error) {
 			break
 		}
 	}
+	if len(versionStr) == 0 {
+		return semver.Version{}, fmt.Errorf("did not find version in command output")
+	}
+
+	// The OSE version will have > 4 parts to the version string
+	// We'll only take the first 3
+	parts := strings.Split(versionStr, ".")
```

Comment:
That's not what I want. SplitN will give me the last segment including the dot. Here I want to remove 4th segment.